### PR TITLE
Stage up

### PR DESCRIPTION
**Pull request by AmjadLille**

### CODE REVIEW

**Review by Guy-deMessimy — DISMISSED** [review]

**Review by Guy-deMessimy — DISMISSED** [review]

**Review by Guy-deMessimy — DISMISSED** [review]

File: src/components/Header/Navbar/index.jsx [hunk 1]
```diff
@@ -11,23 +11,21 @@ export default function Navbar() {
...
             <a href="/engagements">Notre Engagement</a>
-            <a href="/about">Notre Equipe</a>
+            <Link to="/about/#equipe">Notre Equipe</Link>
             <a href="/partenaires">Nos Partenaires</a>
           </DropdownItems>
         </NavDropdown>
         |
         <NavDropdown>
-          <a href="/actions">Nos Actions</a>
+          <a href="#!">Nos Actions</a>
```
Comment (LoicBrassart) [inline src/components/Header/Navbar/index.jsx]: What's that ?

File: src/components/Header/Navbar/index.jsx [hunk 2]
```diff
@@ -11,23 +11,21 @@ export default function Navbar() {
 
           <DropdownItems>
             <a href="/engagements">Notre Engagement</a>
-            <a href="/about">Notre Equipe</a>
+            <Link to="/about/#equipe">Notre Equipe</Link>
```
Comment (LoicBrassart) [inline src/components/Header/Navbar/index.jsx]: Useless `#` ?

File: src/App.js [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
+import { Reset } from 'styled-reset';
+import Contacter from './pages/nous-contacter/index';
+import About from './pages/about/index';
+import Footer from './components/Footer/index';
+import Header from './components/Header/index';
+import StageUp from './pages/stageUp/index';
+import Pedagogie from './components/Pedagogie/index';
+import Atelier from './components/Atelier';
+import Home from './pages/home';
+import { AppStyle } from './style.jsx';
```
Comment (LoicBrassart) [inline src/App.js]: Prettify

File: src/components/Title/style.js [hunk 4]
```diff
@@ -1,10 +1,10 @@
-import styled from "styled-components";
+import styled from 'styled-components';
```
Comment (LoicBrassart) [inline src/components/Title/style.js]: Prettify

File: src/components/Title/style.js [hunk 5]
```diff
@@ -1,10 +1,10 @@
...
 
 const TitleStyle = styled.div`
   display: flex;
   flex-direction: row;
   justify-content: flex-start;
-  font-family: "Roboto", sans-serif;
+  font-family: 'Roboto', sans-serif;
```
Comment (LoicBrassart) [inline src/components/Title/style.js]: Prettify

File: src/style.jsx [hunk 6]
```diff
@@ -1,6 +1,6 @@
-import styled from "styled-components";
+import styled from 'styled-components';
```
Comment (LoicBrassart) [inline src/style.jsx]: Prettify

File: src/style.jsx [hunk 7]
```diff
@@ -1,6 +1,6 @@
...
-  font-family: "Roboto", sans-serif;
+  font-family: 'Roboto', sans-serif;
```
Comment (LoicBrassart) [inline src/style.jsx]: Prettify

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Prettify

**Review by Guy-deMessimy — APPROVED** [review]

**Pull request closed without merging**